src/plots/mapbox/mapbox.js

```diff
@@ -212,7 +212,7 @@ proto.updateData = function(calcData) {
         traceObj = traceHash[trace.uid];
 
         if(traceObj) traceObj.update(calcTrace);
-        else {
+        else if(trace._module) {
```

Comment:
Note that `visible: false` traces do not get `fullTrace._module` during supply defaults.